src/clickhouseEngine.ts

```diff
@@ -200,7 +205,11 @@ export async function clickhouseEngine(): Promise<void> {
 
     // We've reached the end of the view index, so we'll continue but with a
     // delay so as not to thrash the couchdb unnecessarily.
-    if (response.rows.length !== PAGE_SIZE) {
+    if (response.docs.length !== PAGE_SIZE) {
+      i = 0
+      if (lastDocUpdateTime != null) {
+        afterTime = lastDocUpdateTime
```

Comment:
afterTime should be saved in couch or clickhouse somewhere. Otherwise we'll migrate all of couch to clickhouse on reboot of the server.

reports_settings would be a good db in couch to use.

Reply:
makes sense.